frontend/src/components/FriendsListComponent.jsx

```diff
@@ -30,11 +30,7 @@ class FriendListComponent extends Component {
 			return(
 			<div>
 				<i className="server icon"></i>
-				<Link to={
-							{pathname: "/author/"+utils.getStripedEscapedAuthorId(authorObj.url.substring(7,)),
-							  state: {
-							  	fullAuthorId: authorObj.url,
-							  }}}>
+				<Link to={"/author/"+utils.getStripedEscapedAuthorId(authorObj.url.substring(7,))}>
```

Comment:
Zebras have stripes. Our clothes, much like our strings, get stripped. 

```suggestion
				<Link to={"/author/"+utils.getStrippedEscapedAuthorId(authorObj.url.substring(7,))}>
```